app/models/user/User.scala

```diff
@@ -159,6 +159,19 @@ class UserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
       parsed <- Fox.combined(r.toList.map(parse))
     } yield parsed
 
+  def findOwnerByOrg(organizationId: ObjectId)(implicit ctx: DBAccessContext): Fox[User] =
+    for {
+      accessQuery <- accessQueryFromAccessQ(listAccessQ)
+      r <- run(q"""select $columns
+                   from $existingCollectionName
+                   where $accessQuery
+                   and isOrganizationOwner
+                   and not isDeactivated
+                   and _organization = $organizationId
+                   order by _id""".as[UsersRow])
+      parsed <- parseFirst(r, organizationId)
+    } yield parsed
```

Comment:
```suggestion
  def findOwnerByOrg(organizationId: ObjectId): Fox[User] =
    for {
      r <- run(q"""select $columns
                   from $existingCollectionName
                   where isOrganizationOwner
                   and not isDeactivated
                   and _organization = $organizationId
                   order by _id
                   limit 1""".as[UsersRow])
      parsed <- parseFirst(r, organizationId)
    } yield parsed
```
The access query is not needed because everyone who may see the organization json may also fetch the owner. This also makes the DBAccessContext parameter unnecessary.